From: Alberto Villa <avilla@FreeBSD.org>
Date: Sat, 07 Jul 2012 11:31:28 +0000
Subject: Make Nepomuk code rely on NEPOMUK_INCLUDE_DIRS
X-Git-Url: http://quickgit.kde.org/?p=kactivities.git&amp;a=commitdiff&amp;h=2d114b655547ba26254983c1b7f48f8eea058b43
---
Make Nepomuk code rely on NEPOMUK_INCLUDE_DIRS

Including only SOPRANO_INCLUDE_DIR made the build pick up KDE 3
headers on FreeBSD, where KDE SC 4 is installed in non-default prefix
(while Soprano and KDE 3 aren't). NEPOMUK_INCLUDE_DIRS ensures that
KDE include directories come first in the inclusion order.
Additionally, it's better to rely on FindNepomuk.cmake to set the
correct include directories, instead of adding the Soprano one by hand.

This is a backport of commit edbbe3e115b0a4806873969c808383b2926ec581.
---


--- ./service/CMakeLists.txt
+++ ./service/CMakeLists.txt
@@ -31,7 +31,7 @@
 
 if(Nepomuk_FOUND)
     set(HAVE_NEPOMUK 1)
-    include_directories(${SOPRANO_INCLUDE_DIR})
+    include_directories(${NEPOMUK_INCLUDE_DIRS})
     set(ADDITIONAL_LINK_LIBS
         ${ADDITIONAL_LINK_LIBS}
         ${NEPOMUK_LIBRARIES}

